shared/js/initApp.js

```diff
@@ -240,6 +283,9 @@ if (appOptions.droplet) {
         return deferred;
       });
     }
+  } else if (appOptions.level.projectTemplateLevelName) {
+    // this is an embedded project
+    promise.then(dashboard.loadEmbeddedProject(appOptions.level.projectTemplateLevelName));
```

Comment:
If we reject the `$.Deferred` above, will this halt the loading of additional assets?  It looks like the `.then(...)` calls below [only have callbacks registered for success](http://api.jquery.com/deferred.then/).